spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/sql/GcpCloudSqlProperties.java

```diff
@@ -26,12 +26,16 @@
  */
 @ConfigurationProperties("spring.cloud.gcp.sql")
 public class GcpCloudSqlProperties {
+	/** Name of the database in the Cloud SQL instance */
 	private String databaseName;
 
+	/** Cloud SQL instance connection name. [GCP_PROJECT_ID]:[INSTANCE_REGION]:[INSTANCE_NAME] */
 	private String instanceConnectionName;
 
+	/** Overrides the GCP OAuth2 credentials specified in the Core module */
 	private Credentials credentials;
 
+	/** Database type. Either "mysql" or "postgresql" */
```

Comment:
Please don't list available values. Enums are processed automatically and the IDE already offers auto-completion.